test/snippet/alignment/configuration/align_cfg_local_mode_example.cpp

```diff
@@ -1,7 +1,7 @@
-#include <seqan3/alignment/configuration/align_config_mode.hpp>
+#include <seqan3/alignment/configuration/align_config_method.hpp>
 
```

Comment:
shouldn't this file be called `test/snippet/alignment/configuration/align_cfg_local_method_example.cpp `?

Reply:
Yes, but the code in it actually does not make any sense, nor is the file included in the documentation. The snippet is an artifact from the documentation of helper variables for the `mode` method. Sloppy, sorry.